lib/Rex/Commands/Iptables.pm

```diff
@@ -338,13 +372,28 @@ sub is_nat_gateway {
 
     my @iptables_option = ();
 
-    my ($default_line) = run "/sbin/ip r |grep ^default";
+    my ($default_line) = run "/sbin/ip $ip_version r |grep ^default";
     my ($dev)          = ( $default_line =~ m/dev ([a-z0-9]+)/i );
     Rex::Logger::debug("Default GW Device is $dev");
 
-    sysctl "net.ipv4.ip_forward" => 1;
-    iptables t => "nat", A => "POSTROUTING", o => $dev, j => "MASQUERADE";
-
+    if ( $ip_version == -6 ) {
+      sysctl "net.ipv6.conf.all.forwarding",     1;
+      sysctl "net.ipv6.conf.default.forwarding", 1;
+      if ( _iptables_version() < 1_004_018 ) {
+        Rex::Logger::info("NAT for IPv6 supported by iptables >= v1.4.18");
```

Comment:
Is it OK to continue in this case without applying the command in the `else` branch? Why we don't need a `die` in this case?